src/VfacTmdb/Items/TVNetwork.php

```diff
@@ -0,0 +1,134 @@
+<?php declare(strict_types = 1);
+/**
+ * This file is part of the Tmdb package.
+ *
+ * (c) Vincent Faliès <vincent@vfac.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Vincent Faliès <vincent@vfac.fr>
+ * @copyright Copyright (c) 2017-2020
+ */
+
+
+namespace VfacTmdb\Items;
+
+use VfacTmdb\Abstracts\Item;
+use VfacTmdb\Interfaces\Items\TvNetworkInterface;
```

Comment:
Case sensitive problem. It must be 
`use VfacTmdb\Interfaces\Items\TVNetworkInterface;` not `use VfacTmdb\Interfaces\Items\TvNetworkInterface;`

I suppose that you work on Windows OS :)
That's why Scrutinizer controls failed.

Reply:
Aah, good eye! I actually develop on macOS and run the code in an Ubuntu VM. Turns out the file shares will use the case-sensitivity of the host OS (and macOS is case-insensitive by default). Will have to switch the file shares to use NFS. Will fix and commit.